go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/pkg/term v0.0.0-20190109203006-aa71e9d9e942
 	github.com/prometheus/client_golang v1.1.0
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
```

Comment:
Where did this come from? (just wondering)

Reply:
I don't know, it keeps popping up whenever I run `make`; so whichever version of go I have, it thinks it's a dependency.